pennylane/optimize/rotosolve.py

```diff
@@ -55,7 +55,7 @@ class RotosolveOptimizer:
     ...     qml.RX(params[0], wires=0)
     ...     qml.RY(params[1], wires=1)
     ...     qml.CNOT(wires=[0, 1])
-    ...     return qml.expval(qml.PauliX(0)), qml.expval(qml.PauliY(1))
+    ...     return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliX(1))
```

Comment:
(Minor suggestion): on line 64-65 the renamings of `X_1` -> `Z_1` and `Y_2` -> `X_2` could be applied